Sandbox/engine_config.json

```
@@ -0,0 +1,14 @@
+{
+    "WindowSize": [
+        300,
+        150
```

Same as mentioned before.
```suggestion
        1920,
        1080
```